apps/publish/content/common.py

```diff
@@ -723,7 +723,7 @@ def _set_updates_for_media_items(self, doc, updates):
             return
 
         for key in DEFAULT_SCHEMA.keys():
-            if doc.get(key):
+            if doc.get(key) and updates.get("operation") == "publish":
```

Comment:
might be more readable if that condition for publish operation is part of that previous one as it's not really key related so we won't need it in the loop:
```python
if doc.get("type") not in MEDIA_TYPES or updates.get("operation") == "publish":
    return
```